### ISSUE TITLE
Internal Server Error for the version of the app without emojis

### ISSUE THREAD

**Issue by kandrio** [remla23-team06/app#12]
# Overview of the problem

There seems to happen an `Internal Server Error` every time I try to access the application (version without emojis). Here's what happens:

At first, I see the application webpage:

![page](https://github.com/remla23-team06/app/assets/49658958/9d6a1b1b-934a-4056-9277-dc409c099a4c)

I hit submit, and the I see the following error page:

![error](https://github.com/remla23-team06/app/assets/49658958/8bceba95-f697-46dc-93e2-b0c0fcbc783f)

# Trying to debug this

I went and checked the logs of the `app-v1-*` pod using the following command: `kubectl logs app-v1-blah-blah-blah --follow`, and here's what I found:
```
127.0.0.6 - - [08/Jun/2023 11:57:52] "POST /submit HTTP/1.1" 500 -
[2023-06-08 12:03:28,025] ERROR in app: Exception on /submit [POST]
Traceback (most recent call last):
  File "/app/.venv/lib/python3.10/site-packages/requests/models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/.venv/lib/python3.10/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/app/.venv/lib/python3.10/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/app/.venv/lib/python3.10/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/app/.venv/lib/python3.10/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/app/app.py", line 57, in submit
    timeout=1.5).json()
  File "/app/.venv/lib/python3.10/site-packages/requests/models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

# How to reproduce it

Essentially, I did `kubectl apply -f deployment.yml` from the `main` branch of the `operation` repo. The minikube node must have istio, prometheus and grafana installed.

# My reading

It seems like there's an error on the `/submit` path of our app. Probably there's something wrong in the `POST` request we create when we redirect the user to the `/submit` path.

**Comment by Iodine98** [comment]
I ran it using:

```bash
kubectl apply -f deployment.yml
```
from the `main` branch of the `operation` repo. I did pull to the newest version of `main` first though. My minikube node has istio and prometheus installed, not sure about Grafana. There is no internal server error for me, but I do notice when the version with emojis (the one that has version 2.0.0 of the `lib`) redirects immediately to the version without emojis (the one that has version 0.1.1 of `lib`). 

See video below:

https://github.com/remla23-team06/app/assets/12817322/56589362-23d0-4dea-867b-28902a999de4
